projects/demo/src/modules/components/dialog/dialog.module.ts

```diff
@@ -42,6 +42,7 @@ import {SearchDialogExampleModule} from './examples/7/search-example/search-dial
 import {TuiDialogExampleComponent8} from './examples/8';
 import {TuiDialogExampleComponent9} from './examples/9';
 import {PayExampleModalModule} from './examples/9/pay-modal/pay-modal.module';
+import {TuiDialogExampleComponent10} from './examples/10';
```

Comment:
Not sorted

Reply:
lint succeeded because alphabetical sorting works

```
./examples/8/...
./examples/9/...
./examples/9/...
./examples/10/..
```